src/main/java/org/broadinstitute/hellbender/engine/spark/FindAssemblyRegionsSpark.java

```diff
@@ -167,8 +167,14 @@
         // 5. Fill in the reads. Each shard is an assembly region, with its overlapping reads.
         JavaRDD<Shard<GATKRead>> assemblyRegionShardedReads = SparkSharder.shard(ctx, reads, GATKRead.class, header.getSequenceDictionary(), assemblyRegionBoundaries, shardingArgs.readShardSize);
 
-        // 6. Convert shards to assembly regions.
```

Comment:
It looks like you haven't reenabled the downsampling in the first step of the pipeline?

Reply:
It already happens in `getActivityProfileStatesFunction`.